### ISSUE TITLE
Verify v0.1.0 — CI green on 9338aaee + first real four-chain consolidation

### ISSUE THREAD

**Issue by stephenlutar2-hash** [szl-holdings/szl-wave1-report#1]
Verification gate for the v0.1.0 payload (commits `894c201dc28f4707de5d7c083d71dddc465f9044`,
`9338aaee9fd12d44ff9229ea8b2fedda66c8c995`). The suite was executed 9/9 green before the push,
including tamper-detection and fail-closed aggregation paths. Close only with evidence attached.

## Checklist

- [ ] CI workflow run on `9338aaee` is green on Python 3.11 and 3.12 (Actions tab)
- [ ] Fresh clone: `pip install -e . pytest && python -m pytest tests/ -q` → 9 passed
- [ ] Negative control: flip one byte in any receipt in a fixture chain; `aggregate` returns
      `INVALID` naming that harness
- [ ] First real consolidation: after the four harness gates close (szl-calibration#1,
      szl-retrieval-bench#1, szl-engine-bench#1, szl-quant-bench#1), run `aggregate` over the
      four real receipt chains and attach the emitted `wave1-report.md` here
- [ ] The master receipt hash in that report is quoted in `a11oy#1686` when the bakeoff closes

## Non-negotiables

- The README sample output is fixture data, clearly labeled. It is not evidence about any real run.
- A report is publishable only when every input chain verified on the machine that produced it.
- Cross-harness numbers are never ranked against each other — the report enforces this by structure.

**Comment by stephenlutar2-hash** [comment]
### Operator-machine receipt — verifier proven, real four-chain consolidation blocked

Witnessed at exact `main` `5d3c9819b9cdd706b685ca0614ce5c9ea273f6c4`.

- Exact-main CI [run 33891119876](https://github.com/szl-holdings/szl-wave1-report/actions/runs/33891119876): PASS on 3.11/3.12.
- Fresh checkout: 10 tests passed.
- Independent negative control: mutating a quant receipt caused `aggregate` to return `INVALID` naming `szl-quant-bench` and the chain-hash mismatch.
- OPEN/BLOCKED: real engine and real-model receipt chains do not exist, so the four-chain Wave 1 report and its master receipt are unavailable. No fixture master is substituted.

```json
{"chain_hash":"3cb3892401b2a2fa0a8dac8d5dc8c751497c49c6b22927501644cf86927171c8","payload":{"checks":{"exact_main_ci":{"jobs":["pytest (3.11)","pytest (3.12)"],"run":33891119876,"state":"PASS"},"fresh_checkout_tests":{"count":10,"state":"PASS"},"master_receipt":{"state":"UNAVAILABLE"},"real_four_chain_consolidation":{"reason":"upstream real-engine and real-model gates remain open","state":"BLOCKED"},"tamper_negative_control":{"aggregate_state":"INVALID","reason":"szl-quant-bench: payload tampered at receipt 0: chain_hash mismatch","state":"PASS"}},"honesty":"UNSIGNED_HONEST","issue":1,"machine":{"arch":"AMD64","cpu":"Intel Core Ultra 9 285H","gpu":["NVIDIA GeForce RTX 5050 Laptop GPU","Intel Arc 140T GPU"],"os":"Microsoft Windows 11 Home 10.0.26200","python":"3.12.14","ram_gb":31.59},"observed_at":"2026-09-04T16:47:00Z","repo":"szl-holdings/szl-wave1-report","schema":"szl.operator-gate-receipt.v1","source":{"branch":"main","commit":"5d3c9819b9cdd706b685ca0614ce5c9ea273f6c4"}},"prev_hash":"947f07a85f6bf9864110a6819e5066bb4c5204d6d1a548b673a5eba1dded1ca8"}
```

**Comment by stephenlutar2-hash** [comment]
This gate is reopened: no first real four-harness bakeoff closure has been established. The currently open PR #3 upgrades Wave to verify actual native stdlib and calibration receipt schemas, preserve failures, and emit full machine-readable hashes through a real CLI. Tests currently36passed on Python3.11/3.12 after review fixes; publication is not claimed until merged CI readback.

Same-machine real SciFact crosscheck now exists, but the engine first-real-run and estate-model logit criteria remain unavailable. Valid chain integrity or all four supplied filenames do not close those gates. The upgraded report explicitly preserves NOT_EVALUATED acceptance and distinguishes unknown source identity/provenance. A partial measured-source report will be attached separately, not substituted for the required completed bakeoff.

**Comment by stephenlutar2-hash** [comment]
### v0.2.1 installed and merged; fresh partial native report



PR #4 merged as `f5370d5536d0583e94b0ee976e0c12f7d8108b62` after exact-head review and green CI. [Exact-main CI](https://github.com/szl-holdings/szl-wave1-report/actions/runs/34186101029) also passed. 41 tests passed on Python 3.11 and 3.12. All nine local packages are installed at immutable merged source commits.

The old v0.1.0-specific test-count and commit checklist is historical; current-version proof does not retroactively rerun that old version.

This report was generated by installed Wave 0.2.1 from September 8 native receipts: the real SciFact BM25 run (5,183 documents, 300 queries) and an actual loopback calibration request with caller-declared test inputs. Both native chains and the report hash were independently recomputed. Calibration is not real model inference; the retrieval metrics are not an engine benchmark.

The output is one atomic, no-overwrite ZIP containing JSON and Markdown; the legacy two-output rollback race is removed. Source hashes establish integrity, not authenticated issuer identity or independent witnessing.

**Gate stays OPEN.** Real same-model engine endpoints/experiment manifest and estate-model logits remain absent. The first four-harness real bakeoff, acceptance decision, and A11oy closure are not claimed.

# SZL Wave 1 report

Master receipt hash: `c882ce27761ea578c6ddf72c2e70be6c0bbaafdf445ed60f68cdbe5790cd7d0a`

Coverage: INCOMPLETE. Wave 1 acceptance: NOT_EVALUATED.
Hash validity proves integrity, not real-model execution or evidence-gate completion.

Missing harnesses: szl-engine-bench, szl-quant-bench

## Chains

- `szl-calibration` — chain valid (1 receipts; hash); identity HASHED_SOURCE_DECLARATION
- `szl-retrieval-bench` — chain valid (1 receipts; self_hash); identity HASHED_SOURCE_DECLARATION

## Measured (source-reported)

| Harness | Lane | Metrics | Evidence source |
|---|---|---|---|
| szl-calibration | calibration.score.v1 | auroc=1.0, brier=0.009999999999999998, ece=0.09999999999999999, log_loss=0.10536051565782628, mce=0.1, n=2 | CALLER_DECLARED_CALIBRATION_INPUT |
| szl-retrieval-bench | bm25 | P@10=0.0837, R_precision=0.5167, map=0.6003, mrr=0.6086, ndcg@10=0.638, recall@10=0.7605 | UNVERIFIED_SOURCE_RECEIPT |

Cross-harness numbers are never ranked against each other.
Hardware and fairness keys are carried by each source receipt.


<details><summary>Complete machine-readable partial report</summary>

```json
{
  "report_status": "VALID",
  "harnesses": {
    "szl-calibration": {
      "chain": "chain valid (1 receipts; hash)",
      "receipts": 1,
      "identity": "HASHED_SOURCE_DECLARATION"
    },
    "szl-retrieval-bench": {
      "chain": "chain valid (1 receipts; self_hash)",
      "receipts": 1,
      "identity": "HASHED_SOURCE_DECLARATION"
    }
  },
  "coverage_status": "INCOMPLETE",
  "unverified_harness_identities": [],
  "identity_authority": "HASHED_SOURCE_DECLARATION_NOT_ISSUER_AUTHENTICATION",
  "missing_harnesses": [
    "szl-engine-bench",
    "szl-quant-bench"
  ],
  "wave1_acceptance": "NOT_EVALUATED",
  "authority": "INTEGRITY_ONLY_NOT_INDEPENDENT_EVIDENCE",
  "terminal_chain_hashes": {
    "szl-calibration": "cb8b81c5b7a2ca7301a76d347c6f89c7b350169d8b40c52bff160a0132cdf8d3",
    "szl-retrieval-bench": "7ff168030a7c5e404883ed3517afe29885ee14d7a8262284a551a675d0adb473"
  },
  "master_receipt_hash": "c882ce27761ea578c6ddf72c2e70be6c0bbaafdf445ed60f68cdbe5790cd7d0a",
  "measured_lanes": [
    {
      "harness": "szl-calibration",
      "lane": "calibration.score.v1",
      "status": "MEASURED",
      "metrics": {
        "auroc": 1.0,
        "brier": 0.009999999999999998,
        "ece": 0.09999999999999999,
        "log_loss": 0.10536051565782628,
        "mce": 0.1,
        "n": 2
      },
      "detail": "",
      "evidence_source": "CALLER_DECLARED_CALIBRATION_INPUT"
    },
    {
      "harness": "szl-retrieval-bench",
      "lane": "bm25",
      "status": "MEASURED",
      "metrics": {
        "P@10": 0.0837,
        "R_precision": 0.5167,
        "map": 0.6003,
        "mrr": 0.6086,
        "ndcg@10": 0.638,
        "recall@10": 0.7605
      },
      "detail": "",
      "evidence_source": "UNVERIFIED_SOURCE_RECEIPT"
    }
  ],
  "blocked_lanes": [],
  "invalid_lanes": [],
  "unavailable_lanes": [],
  "report_sha256": "4e13039b86bc75b604ce9191d5c2e85c720e21d52ee90ede96d6f1574d8c0d6c"
}
```

</details>
